build.sbt

```diff
@@ -120,7 +120,14 @@ lazy val runtime = CrossProject(
     Test / fork := true
   )
   .jsSettings(
-    scalaJSStage := FastOptStage
+    scalaJSStage := FastOptStage,
+    // While not exactlu ideal, this is only used in the invoker to assign a
+    // unique id to ensure measurements have unique ids. It's never exposed to
+    // the user and doesn't touch anything sensitve, so we should have no
```

Comment:
@ckipp01 unfortunately this is false, this _is_ exposed to the user. If it's a helpful analogy: it would be like if using scoverage on the JVM somehow replaced the JVM's default `SecureRandom` with a fake, insecure one. Since it's a Java API (and not a private internal one) user code is _very_ exposed to this.

I will PR a fix shortly, and I think it would be nice to put out a new RC ASAP. Thanks.

See also:
- https://github.com/scala-js/scala-js/issues/4663

Reply:
Btw, if you really care you should consider putting out a security advisory about this. I think it's pretty unlikely, but in theory if someone is running integration tests with Scala.js against some live infrastructure with scoverage enabled, they could be affected.

Reply:
I'll fix this today, thank you for bringing it to my attention. The RC has very low usage, and it's only been out a day, so I'll include a note about it in the release.

Reply:
Thanks for your attention to this! Much appreciated.

Reply:
For sure... I'll be honest this was me quickly reading and not paying full attention and wanting to quickly get the version bumped without realizing the ramifications. It was foolish of me. I appreciate you bringing it up.